serpentTools/parsers/getmass.py

```diff
@@ -0,0 +1,141 @@
+                    current_a=current_a[-1]
+                elif current_a[0]==' ':
+                    current_a=current_a[1:]
+
+        # now finish the search by moving down one line.
+        l=next(f)
+        l=l.split()[4].split('(')[0]
+        mass=float(l)
+            
+
+
+    return mass 
+
+def getNaturalMass(zaid, z):
+    """gets mass of a natural isotope. this is really meant
+    to just be called by getIsoMass, so please use that function
+    instead. """
+
+    abundances = {}
+    # add abundances by {zaid:abundance} pairs
+    with open(datafile, 'r') as f:
+        current_z='  '
+        while current_z != z:
+            l=next(f)
+            current_z = l[-3:]
+            current_z = current_z[:2] #remove newline
+            current_z = current_z[-1] if current_z[0]==' ' else current_z
+            if z != '1':
+                while l != '\n' :
+                    try:
+                        l=next(f)
+                    except StopIteration:
+                        raise Exception("isotope {} not found".format(zaid))
+
+        while current_z == z:
+            l=next(f)
+            current_z = l[-3:]
+            current_z = current_z[:2] #remove newline
+            current_z = current_z[-1] if current_z[0]==' ' else current_z
+            if z != '1':
+                while l != '\n' :
+                    try:
+                        l=next(f)
+                        
+                        if l[:4] == 'Mass':
+                            current_a = l[-4:]
+                            current_a = current_a[:3] # remove \n
+                            if current_a[:2]=='= ':
+                                current_a=current_a[-1]
+                            elif current_a[0]==' ':
+                                current_a=current_a[1:]
+
+                        lsplit = l.split()
+                        if len ( lsplit ) == 3 or lsplit == []:
+                            continue # isotope does not occur in nature
+                        try:
+                            if lsplit[0]+lsplit[1] == 'IsotopicComposition':
+                                thisZaid = str( int(z)*1000 + int(current_a) )
+                                abundances[thisZaid] = float( lsplit[3].split('(')[0] )
+                        except IndexError:
+                            raise Exception('couldnt find abundance {}'.format(lsplit))
+                    except StopIteration:
+                        raise Exception("isotope {} not found".format(zaid))
+
+    natMass = 0.0
+
+    # add all weighted masses then
+    for key in abundances.keys():
+        natMass += getIsoMass(key) * abundances[key]
+
+    return natMass
+
+# run some examples
+if __name__=='__main__':
+    print('uranium 235: {}'.format(getIsoMass('92235')))
+    print('H {}:'.format(getIsoMass('1001')))
+    print('Kr 73: {}'.format(getIsoMass('36073')))
+    print('Kr 79: {}'.format(getIsoMass('36078')))
+
```

Comment:
Strike this out. 

I like this script. It is very general/not directly tied to SERPENT, but I think it could still benefit the package. We could use this as an command line option, i.e. `python -m serpentTools getIsoMass 92235` and/or some package-level function, i.e. `from serpentTools import getIsoMass`